### PR TITLE
Type conversion: /collect page and all artwork filters

### PR DESCRIPTION
**Pull request by pepopowitz**
This started as a type conversion for the /collect page, and ballooned into several other pages. As I started converting artwork filter components, I decided I should do **all** of them instead of leaving half sans, half serif.

As a result, several pages were all affected by these changes.

## Collect page

All type on this page should now be using new variants. 

local artsy net_5000_collect

## Artist artwork page
local artsy net_5000_artist_pablo-picasso_works-for-sale

## Fairs2 artwork page

cc @mzikherman (because it looks like you've worked on this page pretty recently)

image

## Others?

There are probably other places we have artwork filters, but all filters are accounted for in these three pages. 

## Radio button/checkbox alignment

If I'd only converted the type in radio buttons and checkboxes without making additional padding changes, text was pretty misaligned from the associated control. I added a smidge of top padding to each option to better center the option. I didn't notice until now that by doing so, I also added more space between the options. Take a look and let me know which you'd prefer: 

### 1. without any padding

image

### 2. with padding, but extra space

image

### 3. put some more effort into it Steve, and get it center-aligned without adding space between options (not shown)

### CODE REVIEW

**Comment by willanderson** [comment]
@pepopowitz option 2 (padding, extra space) _looks_ good, and I think it's fine to move forward. Curious if you're using the text component for the labels? Does adjusting the lineheight property to `none` help align things without the extra padding?

**Review by anandaroop — APPROVED** [review]
Nice!

The only other grids I can think of are:
- an individual collection page, which would use this grid
- and the auction page which I think(?) uses a similar but entirely separate grid component, so would be out of scope for this PR

File: src/v2/Components/CollectionsHubsNav.tsx [hunk 1]
```diff
@@ -29,7 +29,11 @@ export const CollectionsHubsNav: FC<CollectionsHubsNavProps> = props => {
           to={`/collection/${hub.slug}`}
           src={resize(hub.thumbnail, { width: 357, height: 175 })}
           ratio={[0.49]}
-          title={<Serif size="4t">{hub.title}</Serif>}
+          title={
+            <Text variant="text" px={1}>
```
Comment (pepopowitz) [inline src/v2/Components/CollectionsHubsNav.tsx:33]: Oh, this `px={1}` is added here to keep the text from wrapping in a weird location. "Impressionist & Modern Art" was wrapping between "Modern" and "Art" after I updated the font. I don't think these 6 hub names are changing any time soon, so adding some padding seemed like an okay way to make it wrap between "&" and "Modern".

**Comment by pepopowitz** [comment]
@willanderson we are using the `Text` component. The issue is that the `Radio` and `Checkbox` components (from palette) nudge the controls down with some top padding. So I'm adding the same padding to the text, to match the control.

**Pull request merged**